api/src/main/proto/stellarstation/api/v1/radio/radio.proto

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2018 Infostellar, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+package stellarstation.api.v1.radio;
+
+option go_package = "radio";
+
+option java_multiple_files = true;
+option java_outer_classname = "RadioProto";
+option java_package = "com.stellarstation.api.v1.radio";
+
+// Messages for modeling radio devices (e.g., receivers, transmitters) in API messages.
+
+// A radio device for RF communication. Usually, a satellite will have one or more radio devices for
+// communicating with the ground and a ground station will have one or more radio devices for
+// communicating with the satellite. Such devices may not actually correspond to actual hardware
+// devices, i.e., in the case of Software Defined Radio (SDR).
+message RadioDevice {
+  // The center frequency of the device, in Hz.
+  uint64 center_frequency_hz = 1;
```

Comment:
If we have `RadioDevice` represent a ground station's radio device, then I'm not sure a single `center_frequency_hz` is appropriate. Ground stations will probably have a range of frequencies they support, unlike satellites which have a few discrete frequencies.

Reply:
Hmm, my impression was that the Lithum radio, for example, corresponds to actual hardware with a range of frequencies it supports. I guess my question would be in those cases, what exactly does the single `center_frequency_hz` represent?

Reply:
Maybe we would rename the message `RadioDeviceConfiguration`

Reply:
Thanks, that actually clears it up great I think

Reply:
Ah, now I get it :)